internal/config/hotkey.go

```diff
@@ -19,6 +19,7 @@ type HotKey struct {
 	ShortCut    string `yaml:"shortCut"`
 	Description string `yaml:"description"`
 	Command     string `yaml:"command"`
+	Navigable   bool   `yaml:"navigable"`
```

Comment:
Navigable feels a bit odd. Not sure if I have a better term tho? 
Was thinking `asBreadCrumb`, 'asTab' or `asChild` or `keepHistory` ??

Reply:
I think `keepHistory` is better :)